docs/source/x_ext.rst

```diff
@@ -338,6 +354,27 @@ An ``id`` ends being in-flight when one of the following scenarios apply:
 * the corresponding commit transaction killed the offloaded instruction and no corresponding memory request transaction and/or corresponding memory result transactions is in progress or still needs to be performed.
 * the corresponding result transaction has been performed.
 
+Superscalar and Multiple Harts
+------------------------------
+
+The interface can be used in scenarios where the |processor| is superscalar, i.e. it can issue more than one instruction per cycle.
+In such scenarios, the the |coprocessor| is usually required to also be able to accept more than one instruction per cycle.
+To support this, the entire interface is replicated according to the issue width. 
```

Comment:
Do you mean the whole "CoreV-X-Interface" and all its sub-interfaces is duplicated ? by `the entire interface is replicated according to the issue width.`
Should issue width be a parameter of the specification **`ISSUE_WIDTH=1`** or **`ISSUE_WIDTH=2`**. 
I don't think duplicating all the interfaces is mandatory.

As an example,  we can have : 
-  2 issue interfaces/ 2 register interfaces
-  2 result interfaces
-  1 memory/memory result interfaces

Or 
-  2 issue interfaces/ 2 register interfaces
-  1 result interfaces
-  0 memory/memory result interfaces

Reply:
parameter added in 4415675